src/Forwarder.cpp

```diff
@@ -172,11 +151,11 @@ std::unique_lock<std::mutex> Forwarder::get_lock_converters() {
 /// Start conversion worker threads, poll for commands from Kafka.
 /// When stop flag raised, clear all workers and streams.
 void Forwarder::forward_epics_to_kafka() {
-  using CLK = std::chrono::steady_clock;
-  using MS = std::chrono::milliseconds;
-  auto Dt = MS(main_opt.MainSettings.MainPollInterval);
-  auto t_lf_last = CLK::now();
-  auto t_status_last = CLK::now() - MS(4000);
+  using STEADY_CLOCK = std::chrono::steady_clock;
+  using MILLISECONDS = std::chrono::milliseconds;
+  auto Dt = MILLISECONDS(main_opt.MainSettings.MainPollInterval);
+  auto TimeSinceLastPoll = STEADY_CLOCK::now();
+  auto TimeSinceLastStatus = STEADY_CLOCK::now() - MILLISECONDS(4000);
```

Comment:
Seeing as you are using chrono literals elsewhere, why not do it here as well:
```suggestion
  using namespace std::chrono_literals; 
  auto TimeSinceLastStatus = STEADY_CLOCK::now() - 4000ms;
```